package.json

```diff
@@ -268,7 +268,7 @@
   "private": true,
   "dependencies": {
     "@department-of-veterans-affairs/formation": "6.9.5",
-    "@department-of-veterans-affairs/formation-react": "5.6.1",
+    "@department-of-veterans-affairs/formation-react": "5.7.0",
```

Comment:
Should be a corresponding yarn.lock change, but I guess that won't exist until the new version of Formation React is published!